src/ui/public/state_management/state_storage/hashing_store.js

```diff
@@ -49,25 +42,12 @@ class HashingStore {
    *  @return {string} the hash of the value
    */
   hashAndSetItem(object) {
-    const json = angular.toJson(object);
+    const json = JSON.stringify(object);
```

Comment:
The current functionality uses angular.toJson for that exact reason and we need to keep using it for now